File: WikipediaDeathsPages.Service/Dtos/DeathDateResultDto.cs
using System;
using System.Collections.Generic;
using Wikimedia.Utilities.Dtos;

namespace WikipediaDeathsPages.Service.Dtos
{
    public class DeathDateResultDto
    {
        public DateTime DateOfDeath { get; set; }
        public int ScoreNumberFour { get; set; }
        public IEnumerable<WikipediaListItemDto> Entries { get; set; }
        public IEnumerable<ExistingEntryDto> RejectedExistingEntries { get; set; }
    }
}

File: WikipediaDeathsPages.Web/ClientApp/src/app/dto/DeathDateResultDto.ts
import { ExistingEntryDto } from './ExistingEntryDto';
import { DeathEntryDto } from "./DeathEntryDto";

export interface DeathDateResultDto{
    dateOfDeath: Date;
    scoreNumberFour: number;
    entries: DeathEntryDto[];
    rejectedExistingEntries: ExistingEntryDto[];
}